src/geometry/geom2/index.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   isA: require('./isA'),
   reverse: require('./reverse'),
   toPoints: require('./toPoints'),
+  toOutlines: require('./toOutlines'),
```

Comment:
hmm After going back & forth a few time I am a bit confused about all our (relatively) similar functions: is the output of 
- toPoints
- toOutlines
- toSides that different ?
Is it not always an array of vec2 ? (sides is a tad different, but even that one has always nagged me a bit)

Reply:
Yeah. It's a tad confusing, and luckly unseeen to most.

- toSides just returns a reference to the data structure (after applying transforms)
- toOutlines returns an array of unique outlines found in the geometry, i.e. edges of holes, etc.
- toPoints returns a array of points (vec2) from the sides, unordered in anyway

We could live without toPoints, but it's being used in test suites.